examples/language/gpt/train_gpt_demo.py

```diff
@@ -309,6 +309,8 @@ def main():
         if n >= WARMUP_STEPS:
             tflops_list.append(step_tflops)
 
+        logger.info(f"max memory {torch.cuda.memory_allocated() / 1024**2} MB", ranks=[0])
```

Comment:
why do we need this line?

Reply:
Yes, I changed it to max_ memory_allocated